docs/src/reference/scripting.md

```diff
@@ -81,6 +81,19 @@ It explains #name.
 Sum is #add(2, 3).
 ```
 
+Let bindings can also be used to unpack arrays. Note that the underscore `_`
```

Comment:
Would it be feasible to extend this to dictionaries and the binding names would have to exactly match the keys? Also: Is the underscore really special? Does `#let _ = 10` not do the same thing?

Reply:
That seems like a useful addition. You're suggesting something like this right?
```
#let (a: x, c: y, ..rest) = ("a": 1, "b": 2, "c": 3, "d": 4)
#x // 1
#y // 2
#rest // ("c": 3, "d": 4)

#let (a: x) = ("z": 1) // error
```
I haven't looked into it, but I think that shouldn't be too hard to implement.

Alright, so the wording there definitely needs improvement. What I meant is that you can do
```
#let (_, _, _) = (1, 2, 3)
#_ // 3
```
But not
```
#let (a, a, a) = (1, 2, 3) // error
```

Reply:
Yep, that's what I'm suggesting. And `a: a` would be the same as `a`.